.github/workflows/find-inactive-tsc.yml

```
@@ -0,0 +1,46 @@
+name: Find inactive TSC members
+
+on:
+  schedule:
+    # Run every Tuesday 12:05 AM UTC.
+    - cron: '5 0 * * 2'
+
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: 16.x
+
+jobs:
+  find:
+
```

We should probably not try to run this on forks, right?
```suggestion
    if: github.repository == 'nodejs/node'
```

---

I must have deleted that line by accident or something. Fixed. Thanks!